test/studies/prk/stencil/EXECOPTS

```diff
@@ -0,0 +1,2 @@
+--iterations=10 --order=1000 --validate
```

Comment:
The inputs are taken from the "small" tests from the PRKs (as opposed to wide). They take <10 seconds. I could shorten them, if we are  targeting <5 second runs.